minvws/nl-kat-coordination
# Limit requesting prior tasks for ranking in scheduler

### Changes

Limit requesting prior task in the `BoefjeScheduler` for ranking. At the moment we don't leverage ranking of tasks based on prior task runs. At least we should limit the number of tasks. This PR will limit this to the last task that has run, since the ranker only uses the last run task at the moment.

### QA notes

Nothing really to QA here. We limit the number of tasks we retrieve from the database to make ranking a bit more optimized. Regular running of scanning jobs should continue to work. Mainly rescanning a job that has already been executed can be tested to make sure that it continues to work.

---

### Code Checklist







---

## Checklist for code reviewers:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_code.md) into your comment.

---

## Checklist for QA:

Copy-paste the checklist from [the docs/source/templates folder](https://github.com/minvws/nl-kat-coordination/blob/main/docs/source/templates/pull_request_template_review_qa.md) into your comment.
